src/vm/eventpipebuffer.h

```diff
@@ -110,8 +149,14 @@ class EventPipeBuffer
     // Get the next event from the buffer, but don't mark it read.
```

Comment:
The "Don't mark it read" of the comment is now confusing. We expect the buffer to be in read-only mode, we are just not logically removing the event from the buffer.